hoomd/mpcd/update.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) 2009-2024 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+r"""Reverse nonequilibrium shear flow of an MPCD fluid.
+
+Reverse nonequilibirum shear flow of an MPCD fluid is established by imposing
+a known momentum flux on the system and measuring the resulting shear rate. This
+provides a way to calculate the shear viscosity of the solvent.
+
+.. invisible-code-block: python
+
+    simulation = hoomd.util.make_example_simulation(mpcd_types=["A"])
+    simulation.operations.integrator = hoomd.mpcd.Integrator(dt=0.1)
+
+"""
+
+from . import _mpcd
+from hoomd.operation import Updater
+from hoomd.data.parameterdicts import ParameterDict
+from hoomd.logging import log
+
+
+class ReverseNonequilibriumShearFlow(Updater):
+    r"""Reverse Perturbation method to establish nonequilibrium shear flow.
+
+    "Craig M. Tenney and Edward J. Maginn. Limitations and recommendations for
+    the calculation of shear viscosity using reverse nonequilibrium molecular
+    dynamics. J. Chem. Phys. 132, 014103, 2010."
+    `<https://doi.org/10.1063/1.3276454>`_
+
+    "Florian Mueller-Plathe. Reversing the perturbation in nonequilibrium
+     molecular dynamics: An easy way to calculate the shear viscosity of
+     fluids. Phys. Rev. E, 59:4894-4898, May 1999."
+     `<https://doi.org/10.1103/PhysRevE.59.4894>`_
```

Comment:
I don't think these need to be written out like this. Just include the DOIs in hyperlinks when you refer to them below.